brave/brave-core
# feat(wallet): Update Assets Filter Options

## Description 
Changes to the `Assets Filter` options

- Removed the `NFTs` option
- Removed the `All Assets` option
- Made `High to Low` option the default
- Made the `High to Low` option the first option above `Low to High`
- Added a `A to Z` and `Z to A` option


Resolves <https://github.com/brave/brave-browser/issues/27323>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

### Testing Ethereum
1. Go to the `Portfolio` tab, the default selection should be `Balance high to low` and filtered correctly
2. Change the filter option to `Balance low to high` the assets should filter correctly.
3. Change the filter option to `Name a to z` the assets should filter correctly.
4. Change the filter option to `Name z to a` the assets should filter correctly.
5. There should no longer be an option to `NFT's and All assets`

https://user-images.githubusercontent.com/40611140/207514665-1771b0e1-0ee7-4691-89e3-f5162c46ad60.mov
